src/viur/core/skeleton.py

```diff
@@ -302,12 +302,29 @@ def __str__(self) -> str:
     def __len__(self) -> int:
         return len(self.boneMap)
 
-    def __ior__self, other: dict) -> None:
-        if not isinstance(other, dict):
-            raise ValueError("Only dictonarys are allowed.")
+    def __ior__(self, other: dict | SkeletonInstance | db.Entity) -> SkeletonInstance:
+        if isinstance(other, dict):
+            for key, value in other.items():
+                self.setBoneValue(key, value)
+        elif isinstance(other, db.Entity):
+            new_entity = self.dbEntity or db.Entity()
+            # We're not overriding the key
+            for key, value in other.items():
+                new_entity[key] = value
+            self.setEntity(new_entity)
+        elif isinstance(other, SkeletonInstance):
+            new_entity = self.dbEntity or db.Entity()
+            # We're not overriding the key
+            for key, value in other.dbEntity.items():
+                new_entity[key] = value
+            self.setEntity(new_entity)
```

Comment:
In this case I would merge the `accessedValues` values from other into self. If other has been changed, but not written this might be different to the dbEntity.